added code in findLocation for coordinate conversion

SMMotteler opened the pull request:
Just added some code to turn coordinates into an address - not a big update yet, but I'm hoping to get CreateGardenActivity to a near-final state by the end of today.

WinnieDaWoo commented:
Nice job on getting this working!

Let me know how your initial debugging og the recycler issue goes, and we will set aside some time as needed to identify the  cause.

For back4app, I think this is an okay workaround for the limitations of the framework. I don't know too much about back4app, but I noticed that there seems to be a `Parse.GeoPoint` API you can use.

`const geopointExample = new Parse.GeoPoint({latitude: 40.0, longitude: -30.0});`

give this a try and let me know how that goes

SMMotteler commented:
![Kapture 2022-06-29 at 12 24 34](https://user-images.githubusercontent.com/68405647/176519462-c18584ee-778d-400f-afff-50d4ddc30871.gif)
It turns out the issue with the fact that clicking my garden wasn't working was because I never set an onclick listener 🙃 so I fixed that and will remove the temp button soon.
I also fixed the fact that my recyclerview for the garden feed would display some gardens two times and others not at all - however, there is still a bug in changing between viewing reminders by time and by garden - I'll check my code for issues with querying.

SMMotteler commented:
the geopoint conversion worked!

WinnieDaWoo reviewed (approved):
code looks good.

Why do you have this in a "example" directory?

SMMotteler commented:
It's how directories have always been generated for me - I didn't know there was any other way.

Also, I'm running into issues with creating my garden in CreateGardenActivity.java (inside com.example.garden_planner) - when I try to add the frost date, it doesn't work, like it never runs into the JSON response handler. I've tried debugging and log statements, but I can't figure anything out. Amy, do you see anywhere I might be going wrong?

![Kapture 2022-06-29 at 16 06 07](https://user-images.githubusercontent.com/68405647/176560466-975c40b0-0432-400e-965a-738f529a839e.gif)

WinnieDaWoo commented:
Can you tell us a little more about the issue? Is it just the frost date that isn't being set? The garden is still being created on click?

I would recommend landing the working stuff you had, and then creating a separate PR for the frost date stuff. That will help isolate the issue.

SMMotteler commented:
got it - thanks for the explanation. I'll make a new branch for the frost date stuff. The garden isn't created because the activity crashes.
